graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLConfiguration.java

```diff
@@ -134,7 +134,7 @@ public static class GraphiQLConfiguration implements Toggleable {
         /**
          * The configuration name of the GraphiQL template path.
          */
-        public static final String TEMPLATE_PATH = PREFIX + ".template-path";
+        public static final String TEMPLATE_PATH_CONFIG = PREFIX + ".template-path";
```

Comment:
why do we need to change this? If we don't need it, lets change it to avoid unnecessary breaking changes.